server/routes/index.js

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import user from './user';
+
+const router = express.Router();
+
+router.use('/', user);
```

Comment:
what is `user`?

Reply:
The user router signifying all routes pertaining to user auth. I initally named it auth.js but the team convention is to name the file user.